doc/antora.yml

```diff
@@ -5,6 +5,11 @@ prerelease: true
 asciidoc:
   attributes:
     cass_url: 'http://cassandra.apache.org/'
+    product: Cassandra
+    cassandra: Apache Cassandra
+    30_version: '3.0'
+    22_version: '2.2'
+    21_version: '2.1'
```

Comment:
what's this section for ?

Reply:
Think of asciidoc attributes as variables that can be used throughout the doc source. The values are substituted into the HTML when rendered. It's a one-to-many use of a string, so that the string only needs replacing in one central file.